drivers/soundwire/stream.c

```diff
@@ -1480,6 +1481,9 @@ static int _sdw_prepare_stream(struct sdw_stream_runtime *stream)
 			return -EINVAL;
 		}
 
+		if (!update_params)
+			goto program_params;
+
```

Comment:
@plbossart looks good only one change needed in the commit message. If INFO_RESUME is not supported, we get the prepare() ioctl followed by TRIGGER_START not TRIGGER_RESUME.

Reply:
Thanks @ranj063, fixed now